kubetest2/internal/deployers/eksapi/nodegroup.go

```diff
@@ -95,7 +95,7 @@ func (m *NodegroupManager) createManagedNodegroup(infra *Infrastructure, cluster
 		ClusterName:   aws.String(m.resourceID),
 		NodegroupName: aws.String(m.resourceID),
 		NodeRole:      aws.String(infra.nodeRole),
-		Subnets:       infra.subnets(),
+		Subnets:       []string{infra.subnetsPrivate[0]},
```

Comment:
AFAIK, efa cannot communicate cross AZ, but should we keep using multi subnets for non efa test case?

Reply:
If the nodes are in two different private subnet, they will also not able to communicate with each other right?

Reply:
I think we should 
if nvidia test: use single az / subnet
if conformance test: using multi az as this change should impact all test cases

Reply:
Don't conformance test communication between nodes?

Reply:
Let me revert this change, and I will implement a backward compatible solution in next PR.